server/index.ts

```diff
@@ -29,7 +30,7 @@ router.get('/', (req, res) => {
   });
 });
 
-router.get('/spaces/poke', async (req, res) => {
+router.get('/spaces/poke', rateLimit, async (req, res) => {
```

Comment:
this `rateLimit` middleware is not for all routes?

Reply:
No, some route don't need it.